test_helpers_end_to_end_ng/src/mini_cluster.rs

```diff
@@ -200,3 +254,108 @@ impl MiniCluster {
         self.other_servers.as_ref()
     }
 }
+
+/// holds shared server processes to share across`
```

Comment:
Looks like an unfinished comment with an extra backtick?

```suggestion
/// holds shared server processes to share across tests
```